SampleScripts/VMWare/VMWare Single Server.ps1

```diff
@@ -0,0 +1,36 @@
+﻿<# Prerequisites :
+    - VMware environment with vCenter server
+    - ResourcePool 'Test'
+    - A VM folder named Templates
+    - In that folder, a powered down VM named 'AL_WindowsServer2012R2DataCenter', fully installed with said OS, and VMware tools installed.\
+    - A snapshot of the VM above (this is to be used as master to the linked clones)
+#>
+
+# Redirect $env:PSmodulepath to develop AutomatedLab modules
+$path = "C:\Users\Jos\Documents\GitHub\AutomatedLab"
+$env:PSModulePath += ";$path"
+
+# Save a credential for VMware access
+$cred = (get-credential administrator@vsphere.local)
+
+# Import VMware modules to current session
+get-module -ListAvailable vmware* | import-module
+
+New-LabDefinition -Name VMWareLab -VmPath C:\AutomatedLab-VMs\ -DefaultVirtualizationEngine VMWare 
+
+Add-LabVMWareSettings -DataCenterName "Datacenter" -DataStoreName datastore1 -VCenterServerName 192.168.1.30 -Credential $cred -ResourcePoolName Test
+
+if (-not (Get-VDPortgroup -Name VMWareLab)){
```

Comment:
Yes, this should ideally be handled in AutomatedLabVMWareWorkerNetwork.psm1 to abstract all network operations and keep AutomatedLab as generic as possible.